3scale-operator/tasks/main.yml

```diff
@@ -47,7 +48,8 @@
           status: "True"
           type: "Available"
         # Usually the APIManager takes 2-3min to get the True status.
-        wait_timeout: 240
+        wait_sleep: 10
+        wait_timeout: 500
       register: result
       failed_when: result.resources[0] is not defined
```

Comment:
this task will fail because there is no `resources` attribute associated with `result`.

If you change this to `failed_when: result is not defined` i think it should give you the behavior you want.

@bcrochet i was under the impression that all objects from kuberntes.core.k8s had a `resources` attribute when calling the registered var...but with this particular one I don't see it...and when i try to call it i get `'dict object' has no attribute 'resources'`....am i missing something?

Reply:
That is correct. I noticed that working on the AAP assessment.